CHANGELOG.md

```diff
@@ -12,6 +12,18 @@ follows:
    releases, and Linux kernel updates. They're also made to fix bugs and add
    features to the build infrastructure.
 
+## v0.5.0
+
+* Changes
+  * Support aarch64 Linux builds
+  * Add libdtc to support runtime loading of device tree overlays
```

Comment:
```suggestion
```

GRiSPv2 doesn't have runtime device tree overlay support. I think that this should be possible, but I didn't enable it since the tools that I've used are bundled with the Raspberry Pi packages.